src/components/CtaItem/CtaItem.jsx

```diff
@@ -0,0 +1,46 @@
+import { Typography } from "@mui/material";
```

Comment:
`for imports`

You can try to sort imports. It will looks like for this file:

import { Typography } from "@mui/material";
import cn from "classnames";

import Btn from "@components/Btn";
import CtaImage from "./CtaImage";

import "./styles.css";

imports from npm
empty line
global imports from (component, or api folder e.t.c)
empty line
local imports
empty line
local styles